product_margin_report_landed_costs/tests/test_product_margin_report_landed_costs.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2025 Binhex <https://www.binhex.cloud>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from unittest.mock import patch
+
+from odoo.tests.common import TransactionCase
```

Comment:
```suggestion
from odoo.addons.base.tests.common import BaseCommon
```
`BaseCommon` is the OCA-recommended approach for tests from 17.0 onward.